testcases/reload-config/reload-config_kw.robot

```diff
@@ -0,0 +1,21 @@
+*** Settings ***
+
+*** Keywords ***
+Connected to cluster-1 
+    Import Library  ${CURDIR}/../../src/KubeLibrary/KubeLibrary.py    kube_config=./cluster1-conf
```

Comment:
It would be best to use KubeLibrary from package here i.e.:
 ```
Import Library    KubeLibrary    kube_config=./cluster1-conf
```

just to be consistent, even knowing it will not work currently.